test/user.test.js

```diff
@@ -214,6 +214,18 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('should invalidate accessToken once user is deleted', function(done) {
+      User.create({ email: 'b@c.com', password: 'bar' }, function(err, userInstance) {
```

Comment:
> userInstance -> instance

@superkhau in the context of this tests, I find the name `instance` confusing. What instance is it - User instance or AccessToken instance?

See https://github.com/strongloop/loopback/pull/2492#discussion_r70971983